spotify_tensorflow/example_decoders_test.py

```diff
@@ -18,7 +18,7 @@
 import json
 
 import tensorflow as tf
-from google.protobuf import text_format
+from google.protobuf import text_format  # type: ignore
```

Comment:
What is this needed for?

Reply:
I don't know. I can jump to that code in IntelliJ just fine, and I've verified that protobuf is installed when mypy runs. I get a fairly generic error:

```
$ tox -e mypy
GLOB sdist-make: /Users/brianm/src/github/spotify/spotify-tensorflow/setup.py
mypy inst-nodeps: /Users/brianm/src/github/spotify/spotify-tensorflow/.tox/.tmp/package/1/spotify_tensorflow-0.5.2.dev15.zip
mypy installed: googleapis-common-protos==1.5.5,mypy==0.650,mypy-extensions==0.4.1,numpy==1.15.4,pandas==0.23.4,protobuf==3.6.1,python-dateutil==2.7.5,pytz==2018.7,six==1.12.0,spotify-tensorflow==0.5.2.dev15,tensorflow-metadata==0.9.0,typed-ast==1.1.0,typing==3.6.6
mypy run-test-pre: PYTHONHASHSEED='1116091400'
mypy runtests: commands[0] | mypy spotify_tensorflow
mypy runtests: commands[1] | mypy tests
tests/example_decoders_test.py:21: error: Module 'google.protobuf' has no attribute 'text_format'
```